src/vs/platform/window/common/window.ts

```diff
@@ -178,6 +179,11 @@ export interface IPathData {
 	// the file path to open within the instance
 	readonly fileUri?: UriComponents;
 
+	/**
+	 * Optional editor options to apply in the file
+	 */
+	readonly options?: IEditorOptions;
```

Comment:
Should also set deprecation to `selection` and `editoOverrideId` here.

Reply:
👍